arcgis-ios-sdk-samples/Route and directions/Offline routing/OfflineRouting.storyboard

```diff
@@ -76,7 +74,8 @@
                                 </items>
                             </toolbar>
                         </subviews>
-                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <viewLayoutGuide key="safeArea" id="4Sz-d2-GIP"/>
+                        <color key="backgroundColor" systemColor="tertiarySystemBackgroundColor"/>
                         <constraints>
                             <constraint firstItem="Ohl-3X-6nD" firstAttribute="top" secondItem="msR-U7-cd1" secondAttribute="bottom" constant="-36" id="00W-hv-SbS"/>
```

Comment:
It seems the constraint here is not correct. Could you also fix it? Thanks.

Reply:
I've made some adjustments. Please let me know what you think, thanks!

Reply:
My apologies, it actually worked correctly. 😅 
But the new constraints also work fine!